config/routes.rb

```diff
@@ -3,7 +3,7 @@
   #api endpoints here
   namespace :api do
     resources 'auth', only: [:create]
-    get 'organizations/:access_token', to: 'auth#organizations'
+    get 'organizations/:access_token', to: 'orgs#get'
```

Comment:
Whenever a `GET` request is made to the `organizations` API, we'll probably want to pass the `access_token` via the `Authorization` header and not through URL params. As an example, the header format will look something like this:

`Authorization` : `token ACCESS_TOKEN`